backstopConfig.js

```diff
@@ -1,56 +1,56 @@
-'use strict';
-// https://github.com/garris/BackstopJS#advanced-scenarios
-
-const backstop = require('@mate-academy/backstop-config');
-const { basicScenario } = backstop;
-
-const basic = {
-  ...basicScenario,
-  label: 'Elementary test',
-  referenceUrl: basicScenario.referenceUrl + '/moyo-header/',
-};
-
-const config = {
-  ...backstop,
-  fileNameTemplate: '{scenarioLabel}_{viewportLabel}',
-  onBeforeScript: 'puppet/onBefore.js',
-  onReadyScript: 'puppet/onReady.js',
-  viewports: [
-    {
-      name: '1024px',
-      width: 1024,
-      height: 768,
-    },
-    {
-      name: '1200px',
-      width: 1200,
-      height: 768,
-    },
-  ],
-  scenarios: [
-    {
-      ...basic,
-      label: 'Header tag',
-      selectors: ['header'],
-    },
-    {
-      ...basic,
-      label: 'Nav tag',
-      selectors: ['nav'],
-    },
-    {
-      ...basic,
-      label: 'Link with data-qa_hover',
-      selectors: ['[data-qa="hover"]'],
```

Comment:
The attribute value should be 'hover' instead of 'data-qa_hover'. Change to 'hoverSelector: '[data-qa="hover"]''.